specs/lightclient/beacon-chain.md

```diff
@@ -189,15 +200,23 @@ def eth2_fast_aggregate_verify(pubkeys: Sequence[BLSPubkey], message: Bytes32, s
 #### `flags_and_numerators`
 
 ```python
-def get_flags_and_numerators() -> Sequence[Tuple[int, int]]:
+def get_flags_and_numerators() -> Sequence[Tuple[ValidatorFlags, int]]:
     return (
         (TIMELY_HEAD_FLAG, TIMELY_HEAD_NUMERATOR),
         (TIMELY_SOURCE_FLAG, TIMELY_SOURCE_NUMERATOR),
         (TIMELY_TARGET_FLAG, TIMELY_TARGET_NUMERATOR)
     )
 ```
 
+```python
+def add_flags(flags: ValidatorFlags, add: ValidatorFlags) -> ValidatorFlags:
```

Comment:
Might be appropriate to name these with `validator` in the name -- `add_validator_flags`. Just because it is typed to `ValidatorFlags` and we might have other flag like things in the spec